web/pdf_sidebar.js

```diff
@@ -104,7 +106,7 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
+      this._hideUINotification(null);
```

Comment:
You're right about that. I'm fine with the removal of `this.close()` as long as no negative side-effects occur, such as an open sidebar where the view is not reset when a document is opened that has no outline/attachments anymore.